### PR TITLE
update info printout cgdfg

### LINKED ISSUE #555: Reintroduce if DFG_USE_CGDFG Environment Variable
This was temporarily removed in #517 but is required for licensing reasons.
https://github.com/JuliaRobotics/DistributedFactorGraphs.jl/blob/c956be5aba890e5a3ef93292a52622b25cd08a45/src/DistributedFactorGraphs.jl#L293

### PR DESCRIPTION
**Pull request by dehann**
Hi Sam, if you wanted to review when you get a chance -- didn't want to bug you on a busy week.  Also see #555

### CODE REVIEW

**Review by GearsAD — APPROVED** [review]

**Comment by dehann** [comment]
thanks!

File: src/DistributedFactorGraphs.jl [hunk 1]
```diff
@@ -291,7 +291,7 @@ function __init__()
     end
 
     # if get(ENV, "DFG_USE_CGDFG", "") == "true"
```
Comment (GearsAD) [inline src/DistributedFactorGraphs.jl:293]: Good point, think we want the if statement as well?

Comment (dehann) [reply src/DistributedFactorGraphs.jl]: that will close #555 too, we can add whenever is good.  Just helping review code while I chew on other stuff.  You should hide and focus on big items for this week.  We can catch up after.

I will add and let you merge when you get a moment.

**Comment by dehann** [comment]
closes #555

File: src/DistributedFactorGraphs.jl (hunk 1 above)
Comment (dehann) [reply src/DistributedFactorGraphs.jl]: ha, thanks!

**Comment by dehann** [comment]
FYI, I also found suggestions to fix #554, see updated first comment in 554

**Pull request merged**